assembl/static2/js/app/graphql/IdeaWithPosts.graphql

```diff
@@ -8,7 +8,7 @@ query IdeaWithPosts($lang: String!, $id: ID!) {
       posts(first: 20) {
         edges {
           node {
-            ... on Post {
+            ... on AssemblPost {
```

Comment:
Why do want to change that? We don't have PropositionPost on an idea, only posts. Post object type can actually be a sqlalchemy AssemblPost, ImportedPost, Email... Maybe this confused you?